apps/src/p5lab/P5Lab.js

```diff
@@ -1468,6 +1469,12 @@ export default class P5Lab {
         this.eventHandlers.draw.apply(null);
       }
     }
+
+    /**
+     * If there is an executionError, alert the user. In Sprite Lab there is
+     * no user-facing console, so we instead create a WorkspaceAlert.
+     */
+    this.alertStudent(this.JSInterpreter.executionError?.message);
```

Comment:
naming nit: this function makes an assumption about what it should do ("if i hit an execution error, alert the student"). there are a few problems with this assumption:
- the user won't always be a student
- changing how we react to execution errors would require changing the same of this function (what if, in the future, instead of alerting, we decide to log to the debug console?)
- this code shouldn't really care how the error is handled, but rather that consumers who _do care_ can handle it however they want